src/aavrutiDJun20/PrimeNumbers.java

```diff
@@ -0,0 +1,49 @@
+package aavrutiDJun20;
+
+public class PrimeNumbers {
+	
+	//Print whether given number is Prime Number or not
+	void primeNumberOrNot(int number) {
+		boolean flag = false;
+		for(int index=2;index<number/2;index++) {
+			if(number%index==0) {
+				flag = true;
+				break;
+			}
+		}
+		if(flag==true)
+			System.out.println(number + " is not a Prime Number");
+		else
+			System.out.println(number + " is a Prime Number");		
+	}
+	
+	//Calculate Sum, Count and Average of Prime Numbers
+	void primeNumber2To100(int startNum, int endNum) {
```

Comment:
Still, method name is not updated.

Reply:
Sorry, Krishna. I missed that one :(
I will keep it in mind from next time